templates/default/sv-sentry-cron-log-run.erb

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
Why do these sv* files show up in the diff?
Anyway you can update the existing services to change the command or what's run based on the sentry version?